### PR TITLE
Created docs for input elements.

### PR DESCRIPTION
**Pull request by Kedar-K**
The docs for the Input Elements has been created. 
1. Proper contents section for each input element added in the documentation.
2. Added the examples of usage of the input components where required so that user can understand better by manipulating it.

### CODE REVIEW

**Comment by Kedar-K** [comment]
@satu0king The formatting of the code and the markdown has been improved so that it has proper syntax and looks clean.

File: _sidebar.md [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-
+* [Home](/)
+* [Getting Started](getting_started.md)
+* [Video Demos](video_demos.md)
+* [Modifying Circuit Elements](Modify.md)
+* [SubCircuits](subcircuit.md)
+* [Exporting Images](export.md)
 
 * Circuit Elements
   * [Standard Gates](gates.md)
```
Comment (satu0king) [inline _sidebar.md]: You need to create an entry here to see the input elements in the website.

Comment (Kedar-K) [reply _sidebar.md]: On it @satu0king. Will commit new changes soon.

Comment (Kedar-K) [reply _sidebar.md]: Changes are made @satu0king.

**Comment by Kedar-K** [comment]
Input Elements entry added under Circuit Elements.

File: _sidebar.md (hunk 1 above)
Comment (satu0king) [reply _sidebar.md]: You might have not pushed the commits @Kedar-K

**Comment by Kedar-K** [comment]
Pushed the commits @satu0king.
you can check it in this [commit 3](https://github.com/CircuitVerse/CircuitVerseDocs/pull/12/commits/2eada9acdcdbae2b44351991bb384fdce6f1028d) at the bottom.

File: inputElements.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+<div>
+    <iframe width="500px" height="400px" src="https://circuitverse.org/simulator/embed/9470" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
+</div>
+
+You can manipulate the above example to see the difference between the Input and ConstantVal and their usage.
+
+## Stepper
+
+The stepper can be used by deagging and dropping the component as shown [above](#getting-started).
```
Comment (satu0king) [inline inputElements.md]: Check dragging spelling

**Comment by satu0king** [comment]
This is a good start. Add your name as a contributor to the page, check how it is done in other pages. I will merge the PR after this.

**Comment by Kedar-K** [comment]
Thank you.
Changes made @satu0king .

**Comment by Kedar-K** [comment]
@satu0king any other changes required?

**Comment by agarwalrounak** [comment]
@Kedar-K I think you can squash the commits into one single commit. It's a good practice.

**Comment by Kedar-K** [comment]
@agarwalrounak ill do it. Thanks.

**Comment by Kedar-K** [comment]
@satu0king any other changes required? And can I start working on documenting output elements or is someone working on that?

File: docs/inputElements.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+In the example above which is **Multi-Input And Gate** you can try changing the input states as per the above instructions.
+
+## Button
+
+A push-Button or simply a **Button** is a simple switch mechanism for controlling some aspect or process. You can use this by simple drag and drop process, it acts as High(1) when pressed and Low(0) when released.
+
+<div>
+    <iframe width="400px" height="400px" src="https://circuitverse.org/simulator/embed/2520" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
+</div>
```
Comment (satu0king) [inline docs/inputElements.md]: This is a bad circuit example.  The other element example are not very good either. You need to explore the various options etc of the elements at least the main ones.

Comment (Kedar-K) [reply docs/inputElements.md]: Ok. I will make changes.

**Comment by Kedar-K** [comment]
@satu0king The changes are made can you please check it.

**Pull request merged**